c/tests/testlib.c

```diff
@@ -136,20 +138,21 @@ const char *nonbinary_ex_sites = "1  0\n"
 const char *nonbinary_ex_mutations = "0    2   1\n"
                                      "1    11  1";
 
-/*** An example of a tree sequence with unary nodes. ***/
+/*** An example of a tree sequence with unary nodes
+ *   and also a non-sample leaf (node 9). ***/
 /*
-0.25┊     8   ┊   8     ┊         ┊
-    ┊   ┏━┻━┓ ┊   ┃     ┊         ┊
-0.20┊   ┃   7 ┊   ┃     ┊   7     ┊
-    ┊   ┃   ┃ ┊   ┃     ┊ ┏━┻━┓   ┊
-0.17┊   6   ┃ ┊   6     ┊ ┃   ┃   ┊
-    ┊ ┏━┻┓  ┃ ┊ ┏━┻━┓   ┊ ┃   ┃   ┊
-0.09┊ ┃  5  ┃ ┊ ┃   5   ┊ ┃   5   ┊
-    ┊ ┃ ┏┻┓ ┃ ┊ ┃ ┏━┻┓  ┊ ┃ ┏━┻┓  ┊
-0.07┊ ┃ ┃ ┃ ┃ ┊ ┃ ┃  4  ┊ ┃ ┃  4  ┊
-    ┊ ┃ ┃ ┃ ┃ ┊ ┃ ┃ ┏┻┓ ┊ ┃ ┃ ┏┻┓ ┊
-0.00┊ 0 1 3 2 ┊ 0 1 2 3 ┊ 0 1 2 3 ┊
-  0.00      2.00      7.00      10.00
+0.25┊      8    ┊   8       ┊           ┊         ┊
+    ┊   ┏━━┻━━┓ ┊   ┃       ┊           ┊         ┊
+0.20┊   ┃     7 ┊   ┃       ┊   7       ┊         ┊
+    ┊   ┃     ┃ ┊   ┃       ┊ ┏━┻━━┓    ┊         ┊
+0.17┊   6     ┃ ┊   6       ┊ ┃    ┃    ┊         ┊
+    ┊ ┏━┻━┓   ┃ ┊ ┏━┻━━┓    ┊ ┃    ┃    ┊         ┊
+0.09┊ ┃   5   ┃ ┊ ┃    5    ┊ ┃    5    ┊         ┊
+    ┊ ┃ ┏━╋━┓ ┃ ┊ ┃ ┏━━╋━━┓ ┊ ┃ ┏━━╋━━┓ ┊         ┊
+0.07┊ ┃ ┃ ┃ ┃ ┃ ┊ ┃ ┃  4  ┃ ┊ ┃ ┃  4  ┃ ┊         ┊
+    ┊ ┃ ┃ ┃ ┃ ┃ ┊ ┃ ┃ ┏┻┓ ┃ ┊ ┃ ┃ ┏┻┓ ┃ ┊         ┊
+0.00┊ 0 1 3 9 2 ┊ 0 1 2 3 9 ┊ 0 1 2 3 9 ┊ 0 1 2 3 ┊
+    0           2           7          10        100
```

Comment:
I've added a non-sample leaf to this example, which we needed to get better coverage.